docs/cloud/index.md

```diff
@@ -0,0 +1,48 @@
+# Feldera Cloud
+
+Feldera Cloud is the enterprise offering of Feldera. It is designed such that
+you can bring your own cloud, on which you deploy Feldera.
+The documentation we provide here is publicly available, both to guide
+our existing customers and to provide insight to interested parties.
+A portion of the Feldera assets (such as container images) required to
+actually perform the steps in this documentation are not publicly
+available, and are only accessible with a license key.
+
+Please contact us at `learnmore@feldera.com` if you are interested to learn
+more or would like a trial. We'd be happy to get you started!
```

Comment:
Shall I fully get rid of "trial" mention, as in reword this to "(...) or would like to try out Feldera Cloud." ?

Reply:
Changed to the above.